### ISSUE TITLE
package named "sir"

### ISSUE THREAD

**Issue by awakumaya** [boutroslab/Supp_Funk_2021#3]
Hi,
Could you tell me how to get package named "sir" in some notebooks of scRNA-Seq-invivo/analysis_pipeline directory.
I guess that the sir package means sir directory in the scRNA-Seq-invivo/analysis_pipeline directory. Is that true? If it's true, how do I set up the directory as a R package?

**Comment by fheigwer** [comment]
The directory you describe does indeed contain an R package that can be installed and contains helper code to support scRNA-Seq analysis. It is maintained by @jan-glx

**Comment by jan-glx** [comment]
That's right, the helper package `sir` is contained in the directory [scRNA-Seq-invivo/analysis_pipeline/sir](/boutroslab/Supp_Funk_2021/tree/main/scRNA-Seq-invivo/analysis_pipeline/sir). 

You can install it from your local clone of the repository by running `make install` within the `analysis_pipeline` directory,

 or, as revealed by the [makefile](https://github.com/boutroslab/Supp_Funk_2021/blob/main/scRNA-Seq-invivo/analysis_pipeline/Makefile) by running `remotes::install_local("sir")` within R while having `analysis_pipeline` as working directory. 
 
 You can also install it directly from github using: `remotes::install_github("boutroslab/Supp_Funk_2021", subdir = "scRNA-Seq-invivo/analysis_pipeline/sir")` ( This may take a minute to download before installation starts since this repo is quite big)
